### PR TITLE
Support array for multipart body

### PR DESCRIPTION
**Pull request by atsushi-ishibashi**
close: #310 
Eventually, I support map and array so keeping backward compatibility.

### CODE REVIEW

File: testdata/book/http_multipart.yml [hunk 1]
```diff
@@ -16,3 +16,14 @@ steps:
...
       current.res.status == 201
+  fileupload2:
+    desc: Post /upload with array
+    req:
+      /upload:
+        post:
+          body:
+            multipart/form-data:
+              - file: ../dummy.png
+              - file: ../dummy.jpg
```
Comment (k1LoW) [inline testdata/book/http_multipart.yml:27]: I think it is good to support multiple files in the following way.
The reason is that the format of inputs other than files remains the same.

```suggestion
              file: 
                - ../dummy.png
                - ../dummy.jpg
```

**Review by k1LoW — COMMENTED** [review]
@atsushi-ishibashi Thank you for your contribution!!

I have one difficult suggestion. Please review.

**Comment by atsushi-ishibashi** [comment]
parse as `map[file:[../dummy.png ../dummy.jpg]]`

**Comment by atsushi-ishibashi** [comment]
seem to need re-run
```
--- FAIL: TestRunUsingSSHd (6.40s)
    --- FAIL: TestRunUsingSSHd/testdata/book/sshd_keep_session.yml (0.08s)
        integration_test.go:101: failed to run testdata/book/sshd_keep_session.yml: test failed on 'Test using SSHd'.steps.get: (current.stdout contains 'fuga') is not true
            current.stdout contains 'fuga'
            ├── current.stdout => ""
            └── "fuga" => "fuga"
```

File: testdata/http_multipart.yml.runbook.golden [hunk 2]
```diff
@@ -17,3 +17,15 @@ steps:
     && current.res.headers['Content-Type'][0] == "text/html; charset=utf-8"
     && 'Date' in current.res.headers
     && current.res.rawBody == "<h1>Posted</h1>"
+- req:
+    /upload:
+      post:
+        body:
+          multipart/form-data:
+            file: dummy.jpg
```
Comment (k1LoW) [inline testdata/http_multipart.yml.runbook.golden:25]: fmfm.
I'd like to fix the one in capture/runbook.go myself!

File: testdata/book/http_multipart.yml [hunk 3]
```diff
@@ -16,3 +16,15 @@ steps:
...
+  fileupload2:
+    desc: Post /upload with array
+    req:
+      /upload:
+        post:
+          body:
+            multipart/form-data:
+              file: 
+                - ../dummy.png
+                - ../dummy.jpg
```
Comment (k1LoW) [inline testdata/book/http_multipart.yml:28]: GREAT!!!

**Review by k1LoW — APPROVED** [review]
LGTM! Thank you!!!!!

**Pull request merged**